mteb/abstasks/GermanRetrievalTask.py

```diff
@@ -0,0 +1,37 @@
+import os
+from .AbsTask import AbsTask
+
+
+class GermanRetrievalTask(AbsTask):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def load_data(self, eval_splits=None, **kwargs):
+        """
+        Load dataset in BeIR format.
+        """
+        try:
+            from beir.datasets.data_loader_hf import HFDataLoader
+        except ImportError:
+            raise Exception("Retrieval tasks require beir package. Please install it with `pip install mteb[beir]`")
+
+
+        if self.data_loaded:
+            return
+        if eval_splits is None:
+            eval_splits = self.description["eval_splits"]
+        dataset = self.description["beir_name"]
+
+
+        self.corpus, self.queries, self.relevant_docs = {}, {}, {}
+
+        for split in eval_splits:
```

Comment:
Instead of adding a `GermanRetrievalTask`, could we do sth like this https://github.com/embeddings-benchmark/mteb/blob/38277ae6f76fe2623373a4e040cc6ddaa592dae7/mteb/tasks/Retrieval/CMTEBRetrieval.py#L7 inside of the `GermanQuAD-Retrieval` class and then just use AbsTaskRetrieval?